APIKit/Errors.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+public enum APIKitError: ErrorType {
+    case InvalidRequest
+    case UnexpectedResponse
```

Comment:
It's nice if there is more detailed error cause as a associated value (as noted in [the comment](https://github.com/ishkawa/APIKit/pull/49/files#r33114482)).